src/app/issuer/components/badgeclass-detail/badgeclass-detail.component.ts

```diff
@@ -184,22 +174,24 @@ export class BadgeClassDetailComponent extends BaseAuthenticatedRoutableComponen
 	revokeInstance(instance: BadgeInstance) {
 		this.confirmDialog
 			.openResolveRejectDialog({
-				dialogTitle: 'Warning',
-				dialogBody: `Are you sure you want to revoke <strong>${this.badgeClass.name}</strong> from <strong>${instance.recipientIdentifier}</strong>?`,
-				resolveButtonLabel: 'Revoke Badge',
-				rejectButtonLabel: 'Cancel',
+				dialogTitle: 'Warnung',
```

Comment:
Would it be possible to have those strings in the language string file thingies? Then we don't have to hardcode every possible language, should we at some point also support English or other languages.